Tests/ChannelIntegrationTests.swift

```diff
@@ -232,31 +232,24 @@ class ChannelIntegrationTests: PubNubSwiftChatSDKIntegrationTests {
   }
 
   func testChannel_SendText() throws {
-    var mentionedUsers = MessageMentionedUsers()
-    var referencedChannels = MessageReferencedChannels()
-
-    mentionedUsers[0] = MessageMentionedUser(id: randomString(), name: "User 1")
-    referencedChannels[0] = MessageReferencedChannel(id: randomString(), name: "Channel 1")
-
     let tt = try awaitResultValue {
       channel.sendText(
-        text: "Please welcome @User who joined the @Chann channel!",
+        text: "Some text to send",
         meta: ["a": 123, "b": "someString"],
         shouldStore: true,
-        mentionedUsers: mentionedUsers,
-        referencedChannels: referencedChannels,
+        usersToMention: nil,
```

Comment:
I understand that we don't test deprecated method any more. Wouldn't be beneficial to have also test for deprecated method if we still offer it to clients?

Reply:
Yes, I also added a test for the legacy method